package.json

```diff
@@ -29,6 +29,7 @@
     "babel-plugin-transform-es2015-spread": "^6.22.0",
     "babel-plugin-transform-es2015-template-literals": "^6.8.0",
     "babel-plugin-transform-inline-imports-commonjs": "^1.2.0",
+    "babel-plugin-transform-optional-catch-binding": "^7.0.0-alpha.16",
```

Comment:
I'm not really confident in depending on alpha code. Is there a stable version of this transform?

Reply:
I don't see any stable version on npm and the latest one is already one year old.

Anyway this package is only necessary to build `readable-stream` from scratch. And so is not relevant to the fix.

I'll just drop the commit adding the dependency and create an issue (#392) to upgrade to babel 7 stable as @targos said.